src/components/profile/ProfileService.js

```diff
@@ -165,7 +165,7 @@ goog.require('ga_urlutils_service');
           $http(params).success(function(data, status) {
             // When all the geometry is outside switzerland
             if (data.length == 0) {
-              data = [{alts: {COMB: 0}, dist: 0}];
+              data = [{alts: {}, dist: 0}];
```

Comment:
We already format the data correctly here -> https://github.com/ioda-net/mf-geoadmin3/blob/83fe80faceb4a8c572aa7b1fceb9e0d04de2e1d2/src/components/profile/ProfileService.js#L100

I think it's good to do this only in one place.